web/app/tournament/tourny.py

```diff
@@ -169,3 +172,93 @@ def select_player(tournament_id, tournament_player_id):
         # flash("Could not delete player: {{ tp.name }}")
         return redirect(url_for('.players_index',
             tournament_id=tournament_id))
+
+
+@tournament.route('/<tournament_id>/matches/', methods=['GET', 'POST'])
+def matches_index(tournament_id):
+    if request.method == "GET":
+        matches = Match.query.filter_by(tournament_id=tournament_id)
+        return render_template('tournament_match_index.html',
+        tournament_id=tournament_id,
+        matches=matches)
+
+    if request.method == 'POST':
```

Comment:
I mean, as part of the post body or as query parameters, it would be nice to specify some details of how the pairing should happen:

Mockup of postbody:
{
  tournamentType: SINGLE_ELEMINATION // or Swiss McMahon. etc.
  playerMatchUp: RANDOM (one could also imagine pairing with similar strengths or disparate strengths).
 ... etc.
}